docker/provision/girder.py

```diff
@@ -0,0 +1,63 @@
+import os
+
+import girder_client
+import requests
+
+ADMIN_USER = 'admin'
+ADMIN_PASS = 'letmein'
+
+
+def setGirderSetting(key, value, gc):
+    return gc.put("/system/setting",
+                  parameters={
+                      "key": key,
+                      "value": value
+                  })
+
+
+def createInitialUser(apiUrl):
+    # no try required here, just get a 400 on failure
```

Comment:
can you change this to "no try/except required here". It will be slightly clearer to me.